app/src/main/java/org/fossasia/openevent/general/auth/ProfileFragmentViewModel.kt

```diff
@@ -26,8 +26,8 @@ class ProfileFragmentViewModel(private val authService: AuthService) : ViewModel
                 .doOnSubscribe({
                     progress.value = true
                 }).doFinally {
-                    progress.value = false
-                }.subscribe({ user ->
+            progress.value = false
+        }.subscribe({ user ->
```

Comment:
This is wrongly indented

Reply:
@iamareebjamal It seems to be right. I just run reformat code on whole package.